### PR TITLE
Fix broken links in Javadocs.

### PR DESCRIPTION
**Pull request by awholegunch**
Passed [Docs Develop Build 2](http://builds.cask.co/browse/CDAP-DDB35-2) which builds the Javadocs...

### CODE REVIEW

File: cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java [hunk 1]
```diff
@@ -70,7 +70,7 @@
  * </p>
  * 
  * <p>
- * See the <i><a href="http://cask.co/docs/cdap/current/en/">Cask DAP Developer Guides</a></i>
+ * See the <i><a href="http://cask.co/docs/cdap/current/en/developers-manual/index.html">CDAP Developers' Manual</a></i>
```
Comment (russorat) [inline cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java]: should it be Developer's? Also, that http://cask.co/docs/cdap/current/en/developers-manual/index.html is a 404

File: cdap-api/src/main/java/co/cask/cdap/api/annotation/ProcessInput.java [hunk 2]
```diff
@@ -40,8 +40,9 @@
  * </pre>
  *
  * <p>
- * See the <i><a href="http://cask.co/docs/cdap/current/en/">Cask DAP Developer Guides</a></i>
- * and the CDAP example applications for more information.
+ * See the <i><a href="http://cask.co/docs/cdap/current/en/developers-manual/index.html">CDAP Developers' Manual</a></i>
+ * and the <i><a href="http://cask.co/docs/cdap/current/en/examples-manual/index.html">CDAP examples</a></i>
```
Comment (russorat) [inline cdap-api/src/main/java/co/cask/cdap/api/annotation/ProcessInput.java]: both urls 404

File: cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java (hunk 1 above)
Comment (awholegunch) [inline cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java]: The title is "Developers" plural-possesive: a manual for developers.

I have fixed the URLs. It was the domain that was the problem.

**Comment by russorat** [comment]
LGTM

**Pull request merged**